.github/actions/deploy/deploy.py

```
@@ -0,0 +1,696 @@
+        return operator_path
+
+    def needs_operator_repo(self) -> bool:
+        """Check if we need to clone the operator repository."""
+        return self.args.deploy_pypi_server
+
+    def deploy_operator(self):
+        """Deploy Data Science Pipelines Operator."""
+        print('🔧 Deploying Data Science Pipelines Operator...')
+
+        if not self.operator_repo_path:
+            raise ValueError('Operator repository not cloned')
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)